SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -931,9 +934,10 @@ void FSpatialGDKEditorToolbarModule::StartLocalSpatialDeploymentButtonClicked()
 void FSpatialGDKEditorToolbarModule::StopSpatialDeploymentButtonClicked()
 {
 	AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [this] {
-		const USpatialGDKEditorSettings* Settings = GetDefault<USpatialGDKEditorSettings>();
-		bool bRuntimeShutdown = Settings->bShutdownRuntimeGracefullyOnPIEExit ? LocalDeploymentManager->TryStopLocalDeploymentGracefully()
-																			  : LocalDeploymentManager->TryStopLocalDeployment();
+		const USpatialGDKEditorSettings* SpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
+		bool bRuntimeShutdown = SpatialGDKEditorSettings->bShutdownRuntimeGracefullyOnPIEExit
+									? LocalDeploymentManager->TryStopLocalDeploymentGracefully()
```

Comment:
Is this whitespace created by the autoformatting rules?

Reply:
yes..